2016/dist/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+      <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
+    <title>SETEC 2016</title>
+    <link rel="stylesheet" href="css/style.min.css">
+  </head>
+  <body>
+<<<<<<< HEAD
```

Comment:
You need to resolve the conflict in this file. You can do that removing the incorrect part of the code